### PR TITLE
Recipe for pandas-datareader

### PR DESCRIPTION
**Pull request by mariusvniekerk**
As requested in #2796

### CODE REVIEW

File: recipes/pandas-datareader/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  license_file: LICENSE.md
+  license_family: BSD
+  summary: 'Up to date remote data access for pandas, works for multiple versions of pandas'
+  dev_url: https://github.com/pydata/pandas-datareader
+  doc_url: https://pandas-datareader.readthedocs.io/en/latest/
+
+extra:
+  recipe-maintainers:
+    - mvanniekerk
+    - femtotrader
```
Comment (synapticarbors) [inline recipes/pandas-datareader/meta.yaml]: @femtotrader -- are you ok being added as a co-maintainer on the conda-forge package for pandas-datareader? You were included by @mariusvniekerk.

Comment (femtotrader) [inline recipes/pandas-datareader/meta.yaml]: I'm still here and I'm ok to be a co-maintainer on the conda-forge package for pandas-datareader however I haven't done any conda-forge package previously. Thanks @mariusvniekerk and @synapticarbors 

@davidastephens is also quite active on this project

File: recipes/requests-ftp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  home: https://github.com/Lukasa/requests-ftp
+  license: Apache-2
+  license_file: LICENSE
+  license_family: Apache
+  summary: 'An FTP transport adapter for use with the Python Requests library'
+  dev_url: https://github.com/Lukasa/requests-ftp
+
+extra:
+  recipe-maintainers:
+    - mvanniekerk
```
Comment (jakirkham) [inline recipes/requests-ftp/meta.yaml]: Oops, guessing this was suppose to be you, @mariusvniekerk. Though we should have caught this too. Do you want to send a PR to the feedstock to correct this? Would be happy to merge if you ping me.

File: recipes/pandas-datareader/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  license: BSD-3
+  license_file: LICENSE.md
+  license_family: BSD
+  summary: 'Up to date remote data access for pandas, works for multiple versions of pandas'
+  dev_url: https://github.com/pydata/pandas-datareader
+  doc_url: https://pandas-datareader.readthedocs.io/en/latest/
+
+extra:
+  recipe-maintainers:
+    - mvanniekerk
```
Comment (jakirkham) [inline recipes/pandas-datareader/meta.yaml]: Oops, guessing this was suppose to be you, @mariusvniekerk. Though we should have caught this too. Do you want to send a PR to the feedstock to correct this? Would be happy to merge if you ping me.

**Pull request merged**